projects/angular/src/utils/list-key-navigation/providers/focusable-items.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { InjectionToken, ValueProvider } from '@angular/core';
+
+export const FocusableElementsToken = new InjectionToken('FocusableElements');
+
+export const FocusableElements = {
+  provide: FocusableElementsToken,
+  useValue: [
+    'a[href]',
+    'area[href]',
+    'input:not([disabled])',
+    'button:not([disabled])',
+    'select:not([disabled])',
+    'textarea:not([disabled])',
+    'iframe',
+    'object',
+    'embed',
+    '*[tabindex]', // -1 tabindex is a focusable element and needed for keyboard navigation
+    '*[contenteditable=true]',
+    '[role=button]:not([disabled])',
+  ],
+} as ValueProvider;
```

Comment:
This should be a type annotation instead of a type assertion b/c a type assertion will not generate a compile-time error here if a property is missing/misspelled.

```typescript
export const FOCUSABLE_ELEMENT_SELECTORS_PROVIDER: ValueProvider = { ... };
```